syntax = "proto3";

package cal;
option cc_generic_services = true;

message AddReq {
    int32 num1 = 1;
    int32 num2 = 2;
};

message AddRsp {
    int32 result = 1;
};

//这是一个http请求，它不需要其他字段
message helloReq {
}
message helloRsp{
}


service CalService {
    rpc Add(AddReq) returns (AddRsp);
    rpc Hello(helloReq) returns (helloRsp);
};